File: openapi-client/javascript-closure-angular/API/Client/SendMailAdvBccInner.js
goog.provide('API.Client.SendMailAdv_bcc_inner');

/**
 * An Email Contact
 * @record
 */
API.Client.SendMailAdvBccInner = function() {}

/**
 * The email address
 * @type {!string}
 * @export
 */
API.Client.SendMailAdvBccInner.prototype.email;

/**
 * (optional) Name to use for the BCC contact.
 * @type {!string}
 * @export
 */
API.Client.SendMailAdvBccInner.prototype.name;

File: openapi-client/scalaz/src/main/scala/org/openapitools/client/api/SendMailAdvBccInner.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import SendMailAdvBccInner._

case class SendMailAdvBccInner (
  /* The email address */
  email: String,
/* (optional) Name to use for the BCC contact. */
  name: Option[String])

object SendMailAdvBccInner {
  import DateTimeCodecs._

  implicit val SendMailAdvBccInnerCodecJson: CodecJson[SendMailAdvBccInner] = CodecJson.derive[SendMailAdvBccInner]
  implicit val SendMailAdvBccInnerDecoder: EntityDecoder[SendMailAdvBccInner] = jsonOf[SendMailAdvBccInner]
  implicit val SendMailAdvBccInnerEncoder: EntityEncoder[SendMailAdvBccInner] = jsonEncoderOf[SendMailAdvBccInner]
}
